README.md

```diff
@@ -1,55 +1,54 @@
 # Getting Started with Compute Engine and the Google Python Client Library
 
-Google Compute Engine features a RESTful API that allows developers to
-run virtual machines in the cloud. This sample python command-line application
-demonstrates how to access the Compute Engine API using the Google Python API
-Client Library. For more information about the library, read the
-[Getting Started: Google Python Client Library documentation][1].
+[Google Compute Engine](https://cloud.google.com/compute/) features a [RESTful API](https://cloud.google.com/compute/docs/api/getting-started)
+that allows developers to run virtual machines in the cloud. This sample python command-line application demonstrates how to access the Compute Engine API using the
+[Google Python API Client Library](https://developers.google.com/api-client-library/python/).
 
-To run the demo:
 
-- Update the variables in the settings.json file.
-    - &lt;your-project-id>: your Compute Engine [project ID][2].
-- Update the client_secrets.json file with your client id and secret found in
-  the [Google APIs Console][3].
-- Run the sample using the command:
+## Pre-requisites
 
+1. Create a project on the [Google Developers Console](https://console.developers.google.com) and [enable billing](https://console.developers.google.com/project/_/settings).
+2. Install the [Google Cloud SDK](https://cloud.google.com/sdk/)
+
+```bash
+curl https://sdk.cloud.google.com | bash
+gcloud auth login
+gcloud config set project your-project-id
+```
+3. Install dependencies using [pip](https://pypi.python.org/pypi/pip)
+
+```bash
+pip install -r requirements.txt
```

Comment:
It varies by OS, distribution, and user settings. I think we can give python developers the benefit of the doubt and assume they know how to use their package manager.